Cargo.toml

```diff
@@ -45,6 +45,8 @@ ignore = "0.4.18"
 postgres-types = { version = "0.2.4", features = ["derive"] }
 cron = { version = "0.12.0" }
 bytes = "1.1.0"
+crypto-hash = { version = "0.3.4", default-features = false }
+flate2 = { version = "1.0.27", default-features = false, features = ["zlib-ng"] }
```

Comment:
These new crates are used to decrypt and encrypt a digest set in the cookie for users returning to the backoffice